detector/aws_elb_duplicate_name_test.go

```diff
@@ -0,0 +1,171 @@
+                            "cross_zone_load_balancing": "true",
+                            "dns_name": "test-elb-tf-12345678.us-east-1.elb.amazonaws.com",
+                            "health_check.#": "1",
+                            "health_check.0.healthy_threshold": "10",
+                            "health_check.0.interval": "30",
+                            "health_check.0.target": "TCP:8000",
+                            "health_check.0.timeout": "5",
+                            "health_check.0.unhealthy_threshold": "2",
+                            "id": "test-elb-tf",
+                            "idle_timeout": "60",
+                            "instances.#": "0",
+                            "internal": "false",
+                            "listener.#": "1",
+                            "listener.206423021.instance_port": "8000",
+                            "listener.206423021.instance_protocol": "http",
+                            "listener.206423021.lb_port": "80",
+                            "listener.206423021.lb_protocol": "http",
+                            "listener.206423021.ssl_certificate_id": "",
+                            "name": "test-elb-tf",
+                            "security_groups.#": "1",
+                            "security_groups.3963419045": "sg-1234abcd",
+                            "source_security_group": "988578293/default",
+                            "source_security_group_id": "sg-abcd1234",
+                            "subnets.#": "2",
+                            "subnets.2342537193": "subnet-1234abcd",
+                            "subnets.3798310056": "subnet-abcd1234",
+                            "tags.%": "0",
+                            "zone_id": "KAHN32700A"
+                        }
+                    },
+                    "provider": ""
+                }
+            }
+        }
+    ]
+}
+`,
+			Response: []*elb.LoadBalancerDescription{
+				&elb.LoadBalancerDescription{
+					LoadBalancerName: aws.String("test-elb-tf"),
+				},
+				&elb.LoadBalancerDescription{
+					LoadBalancerName: aws.String("production-elb-tf"),
+				},
+			},
+			Issues: []*issue.Issue{},
+		},
+	}
+
+	for _, tc := range cases {
+		c := config.Init()
+		c.DeepCheck = true
+
+		awsClient := c.NewAwsClient()
+		ctrl := gomock.NewController(t)
+		defer ctrl.Finish()
+		elbmock := mock.NewMockELBAPI(ctrl)
+		if tc.Response != nil {
+			elbmock.EXPECT().DescribeLoadBalancers(&elb.DescribeLoadBalancersInput{}).Return(&elb.DescribeLoadBalancersOutput{
+				LoadBalancerDescriptions: tc.Response,
+			}, nil)
+		}
+		awsClient.Elb = elbmock
+
+		var issues = []*issue.Issue{}
+		TestDetectByCreatorName(
+			"CreateAwsELBDuplicateNameDetector",
+			tc.Src,
+			tc.State,
+			c,
+			awsClient,
+			&issues,
+		)
+
+		if !reflect.DeepEqual(issues, tc.Issues) {
+			t.Fatalf("Bad: %s\nExpected: %s\n\ntestcase: %s", issues, tc.Issues, tc.Name)
```

Comment:
__[gometalinter]__
* [vet] arg issues for printf verb %s of wrong type: []*github.com/wata727/tflint/issue.Issue

<!-- Comment by [SideCI](https://sideci.com) -->